<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSetting</class>
 <widget class="QWidget" name="FormSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>36</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 36pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacerLeft">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacerRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="buttonOk">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonCancel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMusic">
      <attribute name="title">
       <string>音乐</string>
      </attribute>
      <widget class="QListWidget" name="listMusic">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>16</y>
         <width>649</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabInput">
      <attribute name="title">
       <string>输入</string>
      </attribute>
      <widget class="QKeySequenceEdit" name="keyEditRotate">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>120</y>
         <width>169</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>97</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>旋转：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>168</y>
         <width>97</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>左移：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>216</y>
         <width>97</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>右移：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>264</y>
         <width>97</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>下降：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>312</y>
         <width>97</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>暂停：</string>
       </property>
      </widget>
      <widget class="QKeySequenceEdit" name="keyEditMoveLeft">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>168</y>
         <width>169</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QKeySequenceEdit" name="keyEditMoveRight">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>216</y>
         <width>169</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QKeySequenceEdit" name="keyEditDown">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>264</y>
         <width>169</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QKeySequenceEdit" name="keyEditPause">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>312</y>
         <width>169</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabBg">
      <attribute name="title">
       <string>背景</string>
      </attribute>
      <widget class="QPushButton" name="btnBg1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>32</y>
         <width>137</width>
         <height>89</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/assets/bg1.jpg);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBg2">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>32</y>
         <width>137</width>
         <height>89</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/assets/bg2.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBg3">
       <property name="geometry">
        <rect>
         <x>344</x>
         <y>32</y>
         <width>129</width>
         <height>89</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/assets/bg3.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBg4">
       <property name="geometry">
        <rect>
         <x>488</x>
         <y>32</y>
         <width>129</width>
         <height>89</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/assets/bg4.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
